core/src/main/java/org/elasticsearch/rest/action/cat/RestTable.java

```diff
@@ -109,6 +114,31 @@ public static RestResponse buildTextPlainResponse(Table table, RestChannel chann
         return new BytesRestResponse(RestStatus.OK, BytesRestResponse.TEXT_CONTENT_TYPE, bytesOut.bytes());
     }
 
+    static List<Integer> getRowOrder(Table table, RestRequest request) {
```

Comment:
Ah, you'd need to box anyway to do the sorting so far as I know so this isn't worth it I think.